### ISSUE TITLE
Add dockerize to Alpine's package repository

### ISSUE THREAD

**Issue by farshidtz** [jwilder/dockerize#96]
It would be nice if the latest version of dockerize was submitted to Alpine's package repo so that it could be simply installed via the package manager: `apk add --no-cache dockerize`.

The only version that exists in the repository is 0.2.0:
https://pkgs.alpinelinux.org/packages?name=dockerize&branch=&repo=&arch=&maintainer=

**Comment by jwilder** [comment]
I'm not sure who added it initially.  I agree that it would be nice to add it directly.

**Comment by jhult** [comment]
It looks like @kampka added it.

**Comment by kampka** [comment]
I have bumped the alpine package to version 0.6.1

Please note that issues against alpine packages should be filed on bugs.alpinelinux.org.
If @jhult had not mentioned my directly, I would probably never have seen this (but please don't make that workaround a habit).

Also note that dockerize is currently in the testing repository which is only built for Alpine edge and has to be enabled explicitly. I'd be happy to move it to the community repository if someone is willing to take care of the testing, for which I currently don't have the time.

**Comment by mwmahlberg** [comment]
@kampka I’d have a couple of ideas on how to do integration tests, but I assume there is an "Alpine" way to do it. Can you give me a few pointers?

**Comment by kampka** [comment]
@mwmahlberg you typically implement the [check()](https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#check.28.29) function of the [ABUILD](https://git.alpinelinux.org/cgit/aports/tree/testing/dockerize/APKBUILD).  

Since dockerize does not have an integration test suite, maybe this would be a great time to add some to upstream :)

**Comment by powerman** [comment]
@kampka You may like to switch Alpine package to fork https://github.com/powerman/dockerize - it's up-to-date and have tests. :smile: You can read whole story about this fork in https://github.com/jwilder/dockerize/issues/120

**Comment by kampka** [comment]
@powerman I'd rather not to be honest.
I haven't really used dockerize ever and I haven't been an active contributor of Alpine for some time.
Since I consider switching upstream a non-trivial change for many reasons, I don't feel comfortable to just push this through.

If you feel passionate about this, please feel free to address this on the Alpine dev mailing list.
I'd also be happy to give up mainternership of this package to someone who actually cares about it.

**Comment by jwilder** [comment]
@kampka Please do not.  The fork that has been created is quite different than this project.  It also has questionable contributor practices in that anyone can become a contributor if they ask.  Considering this project is very popular and it typically runs as pid 1 in a container, this would open a potentially large security risk for users unknowingly.  See https://github.com/dominictarr/event-stream/issues/116 as a recent instance of this type of risk.

**Comment by kampka** [comment]
@jwilder I am aware of the risk, that is why I rejected powerman's proposal in the first place.
I am not going to object to the Alpine devs appointing a new maintainer either though, as I am currently not able to put in the work myself.

That being said, I am not overly concerned in this case.
For one, the dockerize Alpine package has never made it out of testing, which requires extra steps to explicitly enable. It also means that there is next to no feedback that the package is working correctly since I packaged it which makes me doubt that the package is very popular among Alpine users.
In addition, changes from new contributors will of course have to pass code review by experienced maintainers as well, which has prevented event-stream like problems in the past quite effectively.

Lastly, I believe this discussion (if there is still need to one) should be moved to the alpine-devel mailing list and this issue should be closed.
